### PR TITLE
Add Portuguese Localization

### PR DESCRIPTION
**Pull request by ansh**
Supersedes #2287 with some code formatting changes

All thanks to @maisondasilva

### CODE REVIEW

File: src/locale/languages.ts [hunk 1]
```diff
@@ -31,6 +32,7 @@ export const APP_LANGUAGES: AppLanguageConfig[] = [
   {code2: AppLanguage.hi, name: 'हिंदी'},
   {code2: AppLanguage.ja, name: '日本語'},
   {code2: AppLanguage.ko, name: '한국어'},
+  {code2: AppLanguage.pt, name: 'Portuguese (BR)'},
```
Comment (ansh) [inline src/locale/languages.ts:35]: I added this clarification @maisondasilva hope this is okay!

Comment (maisondasilva) [reply src/locale/languages.ts:35]: Yes, yes, it is possible to add Brazilian Portuguese to the file languages.ts

https://github.com/bluesky-social/social-app/blob/032a5bd8a744d5096938301ac25b709a50a4db1d/src/locale/languages.ts#L429

**Review by maisondasilva — COMMENTED** [review]
It's ready to be merged!

**Review by pfrazee — APPROVED** [review]
We're going to change this to be keyed by `pt-BR` to specify it's brazlian portuguese. After that looks great! thanks @maisondasilva

**Comment by maisondasilva** [comment]
@ansh @pfrazee Thanks for sending this, You can count on me!
What is the ETA for language release?

**Comment by ansh** [comment]
It's been merged so whenever the next release goes out and is approved by the App Store/Play Store!

**Comment by maisondasilva** [comment]
As you already defined it as pt-BR I sent a correction that I would send in the future when this was done, if you could accept or correct it from your side I would appreciate it!

https://github.com/bluesky-social/social-app/pull/2414

**Pull request merged**